docs/_tutorials/persistence-with-queues.md

```diff
@@ -137,29 +134,32 @@ Now it is time to receive the messages sent to your queue.
 You still need a JMS `Connection` just as you did with the producer. With a connection, you then need to create a Session and bind to the Solace message router queue by creating a `MessageConsumer`. This is nearly identical to what was shown in the publish/subscribe tutorial. In this case, create a Session but use the Solace client acknowledgement mode. This allows the consumers to acknowledge each message individually without side-effects. You can learn more about acknowledgement modes in the Establishing Connections sections of [Solace JMS Documentation – Establishing Connections]({{ site.docs-jms-connections }}){:target="_top"}.
 
 ```java
-Session session = connection.createQueueSession(false, SupportedProperty.SOL_CLIENT_ACKNOWLEDGE);
+Session session = connection.createSession(false, Session.CLIENT_ACKNOWLEDGE);
```

Comment:
An alternative approach would be to keep it as CLIENT_ACK and add a link to this explanation which shows a Solace alternative which solves a specific application problem with the specification version of the JMS ack: http://docs.solace.com/API-Developer-Online-Ref-Documentation/jms/com/solacesystems/jms/SupportedProperty.html#SOL_CLIENT_ACKNOWLEDGE